client/src/App.tsx

```diff
@@ -1,33 +1,36 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { io, Socket} from "socket.io-client";
 import './App.css'
+import { useEffect, useRef } from "react";
+
 
 function App() {
-  const [count, setCount] = useState(0)
+  console.log("test");
+  const socketRef = useRef<Socket | null>(null);
```

Comment:
Załatwmy to może tak, jak tłumaczą ludzie od socket.io: https://socket.io/how-to/use-with-react

Tj. osobny plik `socket.ts` w którym inicjalizujemy 'globalny' socket i możemy z niego korzystać potem gdziekolwiek.